recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/displbe.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Display backend
+Requires=proc-xen.mount weston.service
+After=proc-xen.mount weston.service
+
+[Service]
+Type=simple
+Environment="XDG_RUNTIME_DIR=/run/platform/display"
+ExecStartPre=/bin/sleep 1
+ExecStart=/usr/bin/displ_be
+ExecStartPost=/usr/bin/xenstore-write drivers/displbe/status ready
+ExecStop=/usr/bin/xenstore-write drivers/displbe/status dead
+ExecStopPost=/usr/bin/xenstore-write drivers/displbe/status dead
```

Comment:
`ExecStop` and `ExecStopPost` are identical is this intentional?

Reply:
Yes, it is intentional. ExecStop is executed when you intentionally stops service, before systemd kills any remainings. ExecStopPost is a kind of postmortem action executed even after service failed to start or died its main process.

Reply:
So, `ExecStopPost` will be executed anyways, am I right?

Reply:
Yes, it will be executed anyway as a postmortem. But ExecStop will be executed before displbe kill in a proper shutdown case.